pkgs/servers/monitoring/cockpit/default.nix

```diff
@@ -0,0 +1,152 @@
+{ lib
+, stdenv
+, fetchzip
+, fetchurl
+, fetchFromGitHub
+, autoreconfHook
+, bash
+, coreutils
+, docbook_xml_dtd_43
+, docbook_xsl
+, findutils
+, gettext
+, git
+, glib
+, glibc
+, gnused
+, gnutls
+, json-glib
+, krb5
+, libssh
+, libxcrypt
+, libxslt
+, makeWrapper
+, nodejs
+, openssl
+, pam
+, pkg-config
+, polkit
+, python3Packages
+, ripgrep
+, systemd
+, udev
+, xmlto
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cockpit";
+  version = "282.1";
+  enableParallelBuilding = true;
```

Comment:
```suggestion
```

Reply:
I will do this just before merge because it helps a lot to iterate faster

Reply:
Sorry, why not keep it enabled?

Reply:
I moved it some lines below.